packages/graphql/src/schema-model/relationship/model-adapters/RelationshipAdapter.ts

```diff
@@ -97,6 +97,38 @@ export class RelationshipAdapter {
     public findAttribute(name: string): AttributeAdapter | undefined {
         return this.attributes.get(name);
     }
+    /**
+     * translation-only
+     *
+     * @param directed the direction asked during the query, for instance "friends(directed: true)"
+     * @returns the direction to use in the CypherBuilder
+     **/
+    public getCypherDirection(directed?: boolean): "left" | "right" | "undirected" {
```

Comment:
A thing I don't like about this approach is that we are coupling our internal model (schema model) to the cypher-builder API

It is not a huge problem, as the API is still somewhat agnostic and we own both pieces, but it feels like this is the wrong place for this (at least while the adaptor are so coupled with the schema model)

Reply:
I agree, but the plan was to keep these adapters in the schema-model folder only during the refactoring, but then move them outside in a specialized fashion. Happy to delete these adapaters if the transition will not work out.